### ISSUE TITLE
Custom field type and get-record request

### ISSUE THREAD

**Issue by AlexKovynev** [vitmalina/w2ui#771]
Add a field, changing the type to custom, still empty, without code, so that the fault is not on it. Field is not loaded, changing the type to text - loaded.

**Comment by TrevorPT** [comment]
I think I know what this was about as I just hit the same problem.  When I set up a custom field by following your code at http://w2ui.com/web/docs/form/fields-custom and use it in a form by specifying the custom type in a field descriptor like `{ name: 'title', type: 'title' }`, the DOM field is not set to the value from the form's record when the form is initialized.

I traced it to w2form.refresh 

```javascript
for (var f in this.fields) {
    ...
    switch (field.type) {
        case 'text':
            field.el.value = value;
            break;
        ...
        default:
            $(field.el).w2field($.extend({}, field.options, { type: field.type }));
            break;
```
Note that on the 'text' case it sets `field.el.value`; however, on the default case (a custom field), there is no way to set the initial value of the DOM control so it is blank when the form is shown.  There should be a callback here (maybe given in field.options?) that's triggered in the default custom field case, but I couldn't figure out how to add a callback function to field.options for a custom type.
